tree_sitter/__init__.pyi

```diff
@@ -232,6 +232,7 @@ class Tree:
     @property
     def included_ranges(self) -> list[Range]: ...
 
+    # NOTE: deprecated
```

Comment:
As we are in a stub file, `typing_extensions.deprecated` can be used here :)

Reply:
It's more of a "note to self".